core/libraries/plugin_api/EE_Register_Addon.lib.php

```diff
@@ -371,114 +382,114 @@ private static function _get_addon_settings($class_name, array $setup_args)
             'class_name'            => $class_name,
             // the addon slug for use in URLs, etc
             'plugin_slug'           => isset($setup_args['plugin_slug'])
-                ? (string) $setup_args['plugin_slug']
+                ? (string)$setup_args['plugin_slug']
```

Comment:
code sniffer also expects a space after type casting

This can be adjusted in your IDE at:
phpStorm Settings > Editor > Code Style > PHP > Spaces tab > Other section > After type cast

Reply:
taken care of in the privacy data exporter PR